test/midi-util-test.js

```diff
@@ -0,0 +1,320 @@
+	t.equal(noteFromMidiPitch(82), 'a#5' , 'Note for Midi pitch 82 is a#5')
+	t.equal(noteFromMidiPitch(83), 'b5' , 'Note for Midi pitch 83 is b5')
+	t.equal(noteFromMidiPitch(84), 'c6' , 'Note for Midi pitch 84 is c6')
+	t.equal(noteFromMidiPitch(85), 'c#6' , 'Note for Midi pitch 85 is c#6')
+	t.equal(noteFromMidiPitch(86), 'd6' , 'Note for Midi pitch 86 is d6')
+	t.equal(noteFromMidiPitch(87), 'd#6' , 'Note for Midi pitch 87 is d#6')
+	t.equal(noteFromMidiPitch(88), 'e6' , 'Note for Midi pitch 88 is e6')
+	t.equal(noteFromMidiPitch(89), 'f6' , 'Note for Midi pitch 89 is f6')
+	t.equal(noteFromMidiPitch(90), 'f#6' , 'Note for Midi pitch 90 is f#6')
+	t.equal(noteFromMidiPitch(91), 'g6' , 'Note for Midi pitch 91 is g6')
+	t.equal(noteFromMidiPitch(92), 'g#6' , 'Note for Midi pitch 92 is g#6')
+	t.equal(noteFromMidiPitch(93), 'a6' , 'Note for Midi pitch 93 is a6')
+	t.equal(noteFromMidiPitch(94), 'a#6' , 'Note for Midi pitch 94 is a#6')
+	t.equal(noteFromMidiPitch(95), 'b6' , 'Note for Midi pitch 95 is b6')
+	t.equal(noteFromMidiPitch(96), 'c7' , 'Note for Midi pitch 96 is c7')
+	t.equal(noteFromMidiPitch(97), 'c#7' , 'Note for Midi pitch 97 is c#7')
+	t.equal(noteFromMidiPitch(98), 'd7' , 'Note for Midi pitch 98 is d7')
+	t.equal(noteFromMidiPitch(99), 'd#7' , 'Note for Midi pitch 99 is d#7')
+	t.equal(noteFromMidiPitch(100), 'e7' , 'Note for Midi pitch 100 is e7')
+	t.equal(noteFromMidiPitch(101), 'f7' , 'Note for Midi pitch 101 is f7')
+	t.equal(noteFromMidiPitch(102), 'f#7' , 'Note for Midi pitch 102 is f#7')
+	t.equal(noteFromMidiPitch(103), 'g7' , 'Note for Midi pitch 103 is g7')
+	t.equal(noteFromMidiPitch(104), 'g#7' , 'Note for Midi pitch 104 is g#7')
+	t.equal(noteFromMidiPitch(105), 'a7' , 'Note for Midi pitch 105 is a7')
+	t.equal(noteFromMidiPitch(106), 'a#7' , 'Note for Midi pitch 106 is a#7')
+	t.equal(noteFromMidiPitch(107), 'b7' , 'Note for Midi pitch 107 is b7')
+	t.equal(noteFromMidiPitch(108), 'c8' , 'Note for Midi pitch 108 is c8')
+
+	t.end();
+});
+
+test('Midi -> Util -> mpqnFromBpm', function(t) {
+	var mpqnFromBpm = Util.mpqnFromBpm;
+	var mpqn = mpqnFromBpm(120);
+
+	t.equal(mpqn[0], 7, 'mpqnFromBpm returns expected array');
+	t.equal(mpqn[1], 161, 'mpqnFromBpm returns expected array');
+	t.equal(mpqn[2], 32, 'mpqnFromBpm returns expected array');
+
+	t.end();
+});
+
+test('Midi -> Util -> bpmFromMpqn', function(t) {
+	var bpm = Util.bpmFromMpqn(500000);
+
+	t.equal(bpm, 120, 'bpmFromMpqn returns expected BPM');
+
+	t.end();
+});
+
+test('Midi -> Util -> codes2Str', function(t) {
+	t.equal(Util.codes2Str([65, 66, 67]), 'ABC', 'codes2Str returns expected output');
+
+	t.end();
+});
+
+test('Midi -> Util -> str2Bytes', function(t) {
+	t.equal(Util.str2Bytes('c')[0], 12, 'str2Bytes returns expected output');
+
+	t.end();
+});
+
+test('Midi -> Util -> translateTickTime', function(t) {
+	t.equal(Util.translateTickTime(16)[0], 16, 'translateTickTime translates tick to MIDI timestamp as expected');
```

Comment:
Please compare the entire array; this test won't fail if the array is not of the expected length.